pegasus/migrations/123_add_grade_displayweights_to_tutorials.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :displayweight_k5, String
+      add_column :displayweight_middle, String
+      add_column :displayweight_high, String
```

Comment:
I know we've added some metrics for this information.  Are we going to backfill it at all before it goes live, or just let it populate over time?  Can we start these from the existing displayweight stats for the "all" group?

Reply:
We're going to start with the same ranking from the existing `displayweight` column for the "all" group and use it for every single grade band. We'll have the updated rankings for each grade band by the end of the week.

Reply:
Yes, and those values are already added to the gsheet so the data is ready to go.